### ISSUE TITLE
boolean to keep the sign column open at all times

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vim/vim#117]
```
Feature request:

It'd be nice to have an option to keep the sign column open at all times, so 
the window doesn't have to be redrawn when all signs are gone.  This could be a 
boolean like 'autohidesigncol', or (if you can change the width of the sign 
column) it could be a number like 'foldcolumn' is.
```

Original issue reported on code.google.com by `j...@bendyworks.com` on 5 Mar 2013 at 9:05

**Comment by chrisbra** [comment]
patch available at https://groups.google.com/forum/#!msg/vim_dev/CrBId6DRbvo/RafV2hAQUGkJ

**Comment by jakeFinnBob** [comment]
I'd like to see this feature included as well. Used the patch made by @chrisbra and it works quiet well.

**Comment by Delapouite** [comment]
related commit: https://github.com/vim/vim/commit/95ec9d6a6ab3117d60ff638670a803d43974ba51

**Comment by chrisbra** [comment]
fixed by 95ec9d6
